aws/biganimal_AWS_basepolicy.json

```diff
@@ -3,7 +3,10 @@
     "Statement": [
         {
             "Action": [
+                "autoscaling:CreateOrUpdateTags",
+                "autoscaling:DeleteTags",
                 "autoscaling:DescribeAccountLimits",
+                "autoscaling:DescribeTags",
```

Comment:
Those three new policies needs to be added also in this line https://github.com/EnterpriseDB/cloud-utilities/blob/main/aws/biganimal-csp-setup#L242, which is a compacted JSON of the same "pretty print" version that is updated here. The one that is really used by the script is the compacted JSON of that line, this reference is mostly a pretty print version of the same to better check and see what is included there.

Reply:
Thanks @valeriodelsarto. I'll fix this.